tests/app-template/vite.config.mjs

```diff
@@ -1,70 +1,21 @@
 import { defineConfig } from "vite";
-import {
-  resolver,
-  hbs,
-  scripts,
-  templateTag,
-  optimizeDeps,
-  compatPrebuild,
-  assets,
-  contentFor,
-} from "@embroider/vite";
-import { resolve } from "path";
+import { extensions, classicEmberSupport, ember } from "@embroider/vite";
 import { babel } from "@rollup/plugin-babel";
 
-export default defineConfig(({ mode }) => {
+export default defineConfig(() => {
   return {
-    cacheDir: resolve("node_modules", ".vite"),
-    resolve: {
-      extensions: [
-        ".mjs",
-        ".gjs",
-        ".js",
-        ".mts",
-        ".gts",
-        ".ts",
-        ".hbs",
-        ".hbs.js",
-        ".json",
-      ],
-    },
     plugins: [
-      hbs(),
-      templateTag(),
-      scripts(),
-      resolver(),
-      compatPrebuild(),
-      assets(),
-      contentFor(),
+      classicEmberSupport(),
+      ember(),
 
       babel({
```

Comment:
Not disagreeing with this choice, but I am curious what the reasoning is behind keeping it a user land plugin?

Reply:
so this one felt a bridge too far for me. Sure we have ember stuff in the babel config but ember doesn't need to be in charge of the babel plugin (I think)

I don't have a very strong feeling on it so if we think it should just go in the ember plugin I'm happy, but I didn't want to hide everything 😂

Reply:
we discussed this again today. We've decided it is best to leave it in the config so that you can customise the babel plugin, it's not really possible to customise config going to other plugins from our plugin 🙈